src/component/FilterSection.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useProduct } from "../context/product-context";
+import "../App.css";
+function FilterSection() {
+  const { state, dispatch } = useProduct();
+  console.log(state);
```

Comment:
remove the log

Reply:
Thank you Krituraj for pointing out Removed the logs